### PR TITLE
About

### PR DESCRIPTION
**Pull request by Joephy527**
image
image
image
image
image
image
image
image
image
image

### CODE REVIEW

File: app/about/page.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import NavBar from "../_components/NavBar";
+import AboutHero from "../_components/about/AboutHero";
+import AboutInfo from "../_components/about/AboutInfo";
+import MyStory from "../_components/about/MyStory";
+
+const page = () => {
+  return (
+    <main>
+      <NavBar />
```
Comment (BrookW-83) [inline app/about/page.tsx:9]: -remove all the white spaces inbtween

File: app/page.tsx [hunk 2]
```diff
@@ -8,13 +8,18 @@ import NavBar from "./_components/NavBar";
 export default function Home() {
   return (
     <main>
-      <div className="bg-[url(/images/istockphoto-1311511363-612x612.jpg)] bg-no-repeat bg-cover">
-        <NavBar isHome={true} />
+      <div className="bg-[url(/images/istockphoto-1311511363-612x612.jpg)] bg-no-repeat bg-cover min-h-screen">
+        <NavBar />
```
Comment (BrookW-83) [inline app/page.tsx:12]: - remove all the white spaces inbetween

File: app/_components/(DentalWorks)/Card.tsx
Comment (BrookW-83) [inline app/_components/(DentalWorks)/Card.tsx:1]: - rename the Card to more discriptive name

File: app/_components/Cards.tsx
Comment (BrookW-83) [inline app/_components/Cards.tsx:1]: - use arro function
- rename to more discriptive name

**Review by BrookW-83 — CHANGES_REQUESTED** [review]
- Add all the components in a specific commponents

File: app/about/page.tsx (hunk 1 above)
Comment (Joephy527) [reply app/about/page.tsx:9]: intentional line break between sibling elements for code readability

**Comment by Joephy527** [comment]
put all components in a specific folders, change card component names to more descriptive names, change all functions to arrow functions

**Comment by Joephy527** [comment]
resolve merge conflict, waiting for approval to merge

**Review by BrookW-83 — APPROVED** [review]

**Pull request merged**